clcache.py

```diff
@@ -163,6 +163,9 @@ def clean(self, stats, maximumSize):
         objectInfos = [(os.stat(fn), fn) for fn in objects]
         objectInfos.sort(key=lambda t: t[0].st_atime)
 
+        # compute real current size to fix up the stored cacheSize
+        currentSize = reduce( lambda x, y: x + y[ 0 ].st_size, objectInfos, 0 )
```

Comment:
There's a common name for the function which reduces a sequence of values using `+`: `sum`. :-)

So how about just

    currentSize = sum(x[0].st_size for x in objectInfos)